apps/website/src/pages/docs/alert.mdx

```diff
@@ -17,71 +17,51 @@ import * as AllExamples from 'examples';
 
-Use this only when the system is broken. Leave the technical details behind and explain the problem to the user in an easily understood tone. Mention when the problem might get solved, if it applies.
+By passing the `onClick` prop to `<Alert.Action>` or `<Alert.CloseButton>`, the user can interact with the alert to show additional information or to close. A console message can be logged for event response.
 
-<LiveExample src='Alert.negative.tsx'>
-  <AllExamples.AlertNegativeExample client:load />
+<LiveExample src='Alert.types.tsx'>
+  <AllExamples.AlertTypesExample client:load />
 </LiveExample>
 
-### Multilines
-
-All alerts may support multiple lines if necessary. We recommend keeping it brief and keep the message as concise as possible.
-
-## Placement
+## Variants
 
-The alert notification can display in two different ways:
+### Sticky Alert
 
-### Inline
-
-An inline alert appears embedded within the copy of the site as a block element. This means that when the user scrolls, the alert scrolls as well.
+An alert can be fixed at the top of a user's screen, even if they scroll down the page by passing the `isSticky` prop to the `<Alert.Wrapper>` component. It's important to note that sticky alerts must always be dismissible. Additionally, the alert will appear above all other UI elements except for toast notifications and modals.
 
-<LiveExample src='Alert.inline.tsx'>
-  <AllExamples.AlertInlineExample client:load />
+<LiveExample src='Alert.sticky.tsx'>
+  <AllExamples.AlertStickyExample client:load />
 </LiveExample>
 
-Inline alerts can optionally be dismissible. Once the alert is dismissed, the content below it will shift upwards due to the alert no longer consuming space.
+It is up to the development team wether or not the sticky alert should push the rest of the content down or not.
 
-### Sticky
+### Inline
 
-A sticky alert appears full width stuck at the top of the window frame. That means if the user scrolls, the alert stays at the top of the frame and as the content scrolls underneath it. A sticky alert must always be dismissible. The alert appears above all other UI elements except for toast notifications and modals.
+All alerts support inline placement. Inline alerts can optionally be dismissible. Once the alert is dismissed, the content below it will shift upwards due to the alert no longer consuming space.
 
-<LiveExample src='Alert.sticky.tsx'>
-  <AllExamples.AlertStickyExample client:load />
-</LiveExample>
+### Multilines
 
-It is up to the development team wether or not the sticky alert should push the rest of the content down or not.
+All alerts may support multiple lines if necessary. We recommend keeping it brief and keep the message as concise as possible.
```

Comment:
Like Mayank mentioned, this can be removed. Or it can be combined with the part above that mentions that we recommend a maximum of 256 characters